.rubocop.yml

```diff
@@ -1,5 +1,5 @@
 AllCops:
-  TargetRubyVersion: 2.4
+  TargetRubyVersion: 2.5
```

Comment:
散らばるので .ruby-version で指定したほうが良さそうです

https://github.com/bbatsov/rubocop/blob/master/manual/configuration.md#setting-the-target-ruby-version

> If .ruby-version exists in the directory RuboCop is invoked in, RuboCop will use the version specified by it. Otherwise, users may let RuboCop know the oldest version of Ruby which your project supports with:

https://docs.travis-ci.com/user/languages/ruby/#Using-.ruby-version
https://github.com/nnh/eclipro/blob/master/.travis.yml#L3